test/es7/asyncawait-functionality.js

```diff
@@ -675,6 +676,85 @@ var tests = [
             });  
         }
     },
+    {
```

Comment:
How about with force undo defer?

---
In reply to: [75369702](https://github.com/Microsoft/ChakraCore/pull/1456#discussion_r75369702) [](ancestors = 75369702)